src/app/terra-components.module.ts

```diff
@@ -352,6 +355,7 @@ function createCompiler(compilerFactory:CompilerFactory):Compiler
         TerraDynamicFormComponent,
         TerraDynamicSwitchComponent,
         TerraCategoryPickerComponent,
+        TerraNestedDataPickerComponent,
```

Comment:
You also need to add an export of the TerraNestedDataPickerComponent to the index.ts